content/06.methods.md

```diff
@@ -500,7 +500,7 @@ Next, we used multivariate Cox (proportional hazards) regression analysis [@doi:
 | Software and algorithms                       |                                                              |                                                                                                               |
 | Data processing and analysis software         | Multiple                                                     | See **Table S5** for identifiers                                                                              |
 | OpenPBTA workflows repository                 | this project                                                 | [@doi:10.5281/zenodo.6968175]                                                                                 |
-| OpenPBTA analysis repository                  | this project                                                 |                                                                                                               |
+| OpenPBTA analysis repository                  | this project                                                 | [@doi:10.5281/zenodo.7044567]                                                                                                           |
 | OpenPBTA manuscript repository                | this project                                                 |                                                                                                               |
```

Comment:
should we take this out if not adding a DOI? we have the link at the beginning of the methods